src/common/utils.tsx

```diff
@@ -473,3 +474,11 @@ export function getNextPageParam(options?: { limit: number; offset: number; tota
   if (Math.abs(delta) === sum || isAtEnd) return undefined;
   return sum;
 }
+
+export const getCustomNetworksFromLS = () => {
+  try {
+    return JSON.parse(localStorage.getItem(CustomNetworksLSKey) || '{}') as Record<string, Network>;
+  } catch (_) {
+    return {};
```

Comment:
This would only throw if the item saved in LS were invalid JSON, right? In the error case, would we want to clear the LS item?

Reply:
I added try/catch primarily to not throw if localStorage is undefined